tests/features/splash_screen.feature

```diff
@@ -25,19 +24,19 @@ Feature: Splash Screen features
     And I should see the link "Français"
     And I should see the link "Български"
 
-  @api
-  Scenario: Links on splash screen pages are correct
+  # Regression test for a bug that broke the Portuguese (pt-pt) link.
+  # See https://webgate.ec.europa.eu/CITnet/jira/browse/NEXTEUROPA-5585
+  Scenario: Test language with a custom prefix
     Given I am logged in as a user with the 'administrator' role
-    When I go to "admin/config/regional/language/edit/fr"
-    And I fill in "edit-prefix" with "fr-prefix"
+    And the "pt-pt" language is available
```

Comment:
Because it was not understood by the original reporter that this test covered their use case, since it was not using the language they reported. For me this change is not important at all, but I think people familiar with the issue might search for "pt-pt" in the codebase.